cmd/gcp-controller-manager/app/gke_approver.go

```diff
@@ -477,22 +474,22 @@ func validateTPMAttestation(opts approverOptions, csr *capi.CertificateSigningRe
 		}
 	}
 
-	// TODO(awly): get EK public key from GCE API.
-	// TODO(awly): verify ekCert to root CA.
-	ekCert, err := x509.ParseCertificate(attestCert)
+	// TODO(awly): get AIK public key from GCE API.
+	// TODO(awly): verify aikCert to root CA.
+	aikCert, err := x509.ParseCertificate(attestCert)
 	if err != nil {
 		glog.Errorf("parsing ATTESTATION_CERTIFICATE: %v", err)
 		return false
 	}
-	ekPub, ok := ekCert.PublicKey.(*rsa.PublicKey)
+	aikPub, ok := aikCert.PublicKey.(*rsa.PublicKey)
 	if !ok {
-		glog.Errorf("public key in ATTESTATION CERTIFICATE is %T, want *rsa.PublicKey", ekCert.PublicKey)
+		glog.Errorf("public key in ATTESTATION CERTIFICATE is %T, want *rsa.PublicKey", aikCert.PublicKey)
```

Comment:
nit: capitolize log messages here and below

Reply:
done